src/calculateRentalCost.js

```diff
@@ -27,8 +27,23 @@
  * @return {number}
  */
 
-function calculateRentalCost(days) {
-  // write code here
-}
+const calculateRentalCost = (days) => {
+  const rentalPriceForDay = 40;
+  const minorDiscount = {
+    startDay: 3,
+    value: 20,
+  };
+  const majorDiscount = {
+    startDay: 7,
+    value: 50,
+  };
+  const finalDiscount = days >= majorDiscount.startDay
+    ? majorDiscount.value
+    : days >= minorDiscount.startDay
+      ? minorDiscount.value
+      : 0;
```

Comment:
don't use nested ternaries, remake it with plain ifs where you can return values right away